src/main/resources/config/liquibase/changelog/20210510_transaction_createdBy_column_update.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+    <changeSet author="LUXIANZE" id="20210510_5">
+        <modifyDataType columnName="closed_by" newDataType="long" tableName="transaction"/>
```

Comment:
I tot `closed_by` and `created_by` are based on user `login`, instead of userId??

Reply:
Created_by is, but closed_by is new field, is it better to store the login instead of the user_id directly?

Reply:
yeah, better use login. So that there is no confusion. Every column with suffix `_by` (created_by, closed_by) we standardize to use login. Is it okay?

Reply:
Ok, confirm tonight

Reply:
updated to use closed_by datatype varchar(50)